pkg/env/sensor.go

```diff
@@ -20,6 +20,12 @@ var (
 	// This is typically used for Sensor to communicate with a local Scanner-slim's gRPC server.
 	ScannerSlimGRPCEndpoint = RegisterSetting("ROX_SCANNER_GRPC_ENDPOINT", WithDefault("scanner.stackrox.svc:8443"))
 
+	// ScannerV4GRPCEndpoint is used to communicate with the Scanner V4 endpoint in the same cluster.
+	ScannerV4GRPCEndpoint = RegisterSetting("ROX_SCANNER_V4_GRPC_ENDPOINT", WithDefault("scanner-v4.stackrox.svc:8443"))
+
+	// ScannerV4Enabled Enables Scanner V4 as the default StackRox image scanner in Sensor.
+	ScannerV4Enabled = RegisterBooleanSetting("ROX_SCANNER_V4_ENABLED", false)
```

Comment:
nit: I think just `ROX_SCANNER_V4` may be better, as it is concise and setting it to `true` would imply it's enabled. I do see another env var/feature flag with `_ENABLED`, but I think most just imply it